build/common.props

```diff
@@ -25,8 +25,7 @@
     <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(SrcTargets)</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">$(SrcStandardTargets)</TargetFrameworks>
+    <TargetFrameworks>$(SrcTargets)</TargetFrameworks>
```

Comment:
I'm not sure what was the rationale for using different targets depending on build OS but it doesn't make sense. .NET 8/9 SDK can build all the supported targets.